front/src/modules/trainschedule/components/Timetable/TimetableToolbar.tsx

```diff
@@ -140,100 +139,111 @@ const TimetableToolbar = ({
 
   return (
     <>
-      <div className="scenario-timetable-toolbar justify-content-between">
-        <div className="multi-select-buttons">
-          {multiSelectOn && (
-            <>
-              <input
-                type="checkbox"
-                className="mr-2"
-                checked={selectedTrainIds.length === trainSchedulesWithDetails.length}
+      <div
+        className={cx('scenario-timetable-toolbar', {
+          centered: trainSchedules.length === 0,
+        })}
+      >
+        <div className="toolbar-header">
+          {trainSchedules.length === 0 ? (
+            <div className="no-train">
```

Comment:
Mockup : 
![Capture d’écran 2024-09-24 à 16 30 26](https://github.com/user-attachments/assets/4f8baa08-543b-40af-94d1-113ed94117a9)

PR : 
![Capture d’écran 2024-09-24 à 16 30 18](https://github.com/user-attachments/assets/7d5ee645-bdf4-4b66-861c-829702b72716)

Color is wrong (you need to add `grey80` variable) and it seems like the alignment is not correct

Reply:
Done

Reply:
Seems like you added the grey80 in variables but it's not used and alignment i still not correct

Reply:
Done

Reply:
Ok for the "no train" but when there are trains

Reply:
I've just opened a [bug ticket](https://github.com/OpenRailAssociation/osrd-ui/issues/552) in the `ui` repository. When the bug is solved, the problem should also be solved.